RELEASE/scripts/autoscend/auto_restore.ash

```diff
@@ -1687,7 +1745,7 @@ void __cure_bad_stuff()
 	// let mafia figure out how to best remove beaten up
 	if(have_effect($effect[Beaten Up]) > 0){
 		auto_log_info("Ouch, you got beaten up. Lets get you patched up, if we can.");
-		uneffect($effect[Beaten Up]);
+		auto_beaten_handler();
 
```

Comment:
Fixes auto_beatenUpCount from always being 0 regardless of how many times we get beaten up. Saves having to grep through session logs to determine if maximizer changes in paths like BHY are effective in preventing us getting constantly trounced.